test/setup

```diff
@@ -26,45 +26,29 @@ echo "testenv_instance_prefix: ${testenv_instance_prefix}"
 echo "testenv_heat_stack_name: ${testenv_heat_stack_name}"
 
 echo "create inventory directory"
-rm -rf ${ROOT}/envs/test
-mkdir -p ${ROOT}/envs/test
+rm -rf ${ENV_DIR}
+mkdir -p ${ENV_DIR}
+cp -r ${ROOT}/envs/example/${TEST_ENV}/* ${ENV_DIR}
 
-cp ${ROOT}/envs/example/defaults-2.0.yml ${ROOT}/envs/test/defaults-2.0.yml 
-cat <<eof >> ${ROOT}/envs/test/defaults-2.0.yml 
+cat <<eof >>  ${ENV_DIR}/group_vars/all.yml
 testenv_instance_prefix: ${testenv_instance_prefix}
 testenv_keypair_name: ${KEY_NAME}
 testenv_keypair_path: ${KEY_PATH}
 eof
 
-cp -r ${ROOT}/envs/example/${TEST_ENV}/* ${ROOT}/envs/test
-echo "creating heat template for test env if no heat_stack.yml in envs/test/..."
-if [[ ! -e ${ROOT}/envs/test/heat_stack.yml ]]; then
-  ursula --ursula-debug "envs/test" ${ROOT}/playbooks/${TEST_ENV}/tasks/create_heat_template.yml --extra-vars="use_private_ips=${USE_PRIVATE_IPS}"
```

Comment:
If you want to delete these lines, please make sure the heat_stack.yml of all test_envs is under envs/example/<test_env>/ not playbooks/<test_env>/. Such as "allinone"....